src/client/components/OrdersSummary/OrdersSummary.styles.css

```diff
@@ -0,0 +1,32 @@
+.orders-view .order-item {
+  border: solid white 3px;
+  background-color: #f6f8eb;
+  margin-bottom: 15px;
+}
+
+.orders-view button {
```

Comment:
Nitpick:
Avoid naming your CSS classes after which HTML tag is being used, you might change the tag but the class name should not need to change for that.

Similarly, don't name the class based on what CSS properties it uses, like "flex-container".

Aim instead to name your CSS classes for what the objects on the page mean to the user!